src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+/**
+ * Marks a task identified by its displayed index in the GUI as completed.
+ */
+public class AddTaskCommand extends Command {
+    public static final String COMMAND_WORD = "addTask";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Add a task to a taskList\n"
+            + "Parameters: Task\n"
+            + "Example: " + COMMAND_WORD + " d/<Purchase milk> D/<deadline> t/<tag>";
+    public static final String MESSAGE_ADD_TASK_SUCCESS = "Task added complete: %1$s";
```

Comment:
Tested your code, the output for the Output box is wrong 
![image](https://user-images.githubusercontent.com/56034480/194785704-86a21a09-2d03-4ee4-9855-4561a3bc21be.png)